sdk/eventhub/event-hubs/src/eventHubProducerClient.ts

```diff
@@ -151,12 +161,45 @@ export class EventHubProducerClient {
     return producer.createBatch(options);
   }
 
-  async sendBatch(batch: EventDataBatch, options?: SendOptions): Promise<void>;
-  async sendBatch(batch: EventDataBatch, partitionId: string, options?: SendOptions): Promise<void>;
+  /**
+   * Sends a batch of events to the associated Event Hub.
+   *
+   * @param eventData an instance of `EventDataBatch` (to create a batch use `createBatch`)
```

Comment:
```suggestion
   * @param batch an instance of `EventDataBatch` (to create a batch use `createBatch`)
```

Reply:
Fixed with https://github.com/Azure/azure-sdk-for-js/pull/5964/commits/92eb64d98eb395114ec3f7f761db85106344c2f8